extension/src/popup/Router.tsx

```diff
@@ -277,7 +277,7 @@ const Outlet = () => {
     ((location.pathname === ROUTES.welcome &&
       applicationState === APPLICATION_STATE.MNEMONIC_PHRASE_CONFIRMED) ||
       SHOW_NAV_ROUTES.some((route) => location.pathname === route) ||
-      isSwap);
+      (isSwap && location.pathname !== ROUTES.unlockAccount));
```

Comment:
ugh - sorry for adding even more twisty logic to this conditional: but we're basically saying that if we show the password auth screen during the Swap flow (bc the user's session happened to time out), don't show the bottom nav

Reply:
makes sense!